.tav.yml

```diff
@@ -367,7 +367,7 @@ elasticsearch:
   commands: node test/instrumentation/modules/@elastic/elasticsearch.test.js
 '@elastic/elasticsearch-canary':
   name: '@elastic/elasticsearch-canary'
-  versions: '^8.0.0-canary.35'
+  versions: '>=8.0.0-canary.36 || >=8.1.0-canary.2'
```

Comment:
REVIEW NOTE: Kibana's HEAD is using `8.1.0-canary.2` now. I had to drop at least `8.0.0-canary.35` because our tests depend on https://github.com/elastic/elasticsearch-js/commit/76659b604f97993da84a5e3d12d9df7c3e0cbd52 which special-cased handling of the "sort" param to client commands such that it is always passed as a query param to ES.